src/main/java/com/offdk/play/model/game/MatchPlayer.java

```diff
@@ -7,28 +7,29 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
+import com.offdk.play.model.slack.User;
 import com.offdk.play.rating.RatingType;
 
 import io.vavr.control.Try;
 
 public class MatchPlayer {
-  private final String id;
+  private final User user;
   private final MatchPlayerType type;
   private Optional<Integer> score;
   private Optional<BigDecimal> rating;
 
   // Needed to allow spring-data to serialize data to object
   @PersistenceConstructor
-  MatchPlayer(String id, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
-    this.id = id;
+  MatchPlayer(User user, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
+    this.user = user;
     this.type = type;
     this.score = score;
     this.rating = rating;
   }
 
   @VisibleForTesting
```

Comment:
No, this was needed.  MatchPlayer object is a snapshot of an player (drumroll... just like identity).  A matchPlayer is carries additional information like score and player type (challenger vs challenged).  Rating calculator returns a list of rating, which can be set on player objects separately.

As much as I would like to use player for everything, it just doesn't make sense given what we have to do.

Also, i would like to discuss if we really need to keep multiple ratings per player.  I don't think this is necessary as you will never compare multiple ratings at any given time.  If we decide to switch rating system, i would rather recalculate.